exla/lib/exla/defn.ex

```diff
@@ -733,6 +733,44 @@ defmodule EXLA.Defn do
     )
   end
 
+  defp to_operator(:take_along_axis, [tensor, indices, axis], _ans, state) do
+    indices_shape = op_shape(indices)
+    indices_rank = tuple_size(indices_shape)
+
+    axes_range = 0..(indices_rank-1)
+
+    index_vector_dim = indices_rank
+    slice_sizes = 1 |> List.duplicate(indices_rank)
```

Comment:
You shall never pipe once. :)

```suggestion
    slice_sizes = List.duplicate(1, indices_rank)
```

Reply:
Haha! There's a credo rule for that :rofl:

Reply:
PS: this was just remaining from a refactor, I would never knowingly commit such an atrocity